intg-denonavr/simplecommand.py

```diff
@@ -350,150 +371,150 @@ async def send_simple_command(self, cmd: str) -> ucapi.StatusCodes:
             return await self._handle_audyssey_command(cmd)
         if cmd in DIRAC_COMMANDS:
             return await self._handle_dirac_command(cmd)
-        return ucapi.StatusCodes.NOT_IMPLEMENTED
+
+        # send raw command to the receiver
+        return await self._send_command(cmd)
```

Comment:
We should be cautious sending raw commands from the user to the receiver and only allow command strings that _could_ make sense. If I'm not mistaken all Denon commands are uppercase ASCII 7bit letters, numbers and space.
What do you think about a validation? If any other character is found: bad request.
There's also no URL encoding in DenonDevice._send_command for the provided (user) parameter, except replacing spaces.

Reply:
This is what I could find in their spec:
<img width="777" height="315" alt="image" src="https://github.com/user-attachments/assets/27d8fba6-7baf-42b9-8b93-90a88144ea81" />

Reply:
Adjusted :)